crates/rnote-ui/data/ui/shortcuts.ui

```diff
@@ -39,6 +39,12 @@
                 <property name="accelerator">&lt;ctrl&gt;q</property>
               </object>
             </child>
+            <child>
+              <object class="GtkShortcutsShortcut">
+                <property name="title" translatable="yes">Toggle Tabs Overview</property>
+                <property name="accelerator">&lt;ctrl&gt;&lt;shift&gt;o</property>
+              </object>
+            </child>
```

Comment:
I'd move the position of this button to the second from left - so right of the new-tab button and left of the focus-mode button.